### PR TITLE
Update mocpy maintainer list

### PR DESCRIPTION
**Pull request by pllim**
This is to mirrror a change at astropy/astropy-v5.0-paper#81 . I think we should formalize it here first before putting that on the paper.

- [x] Signed off by @fxpineau
- [x] Signed off by @bmatthieu3

mocpy was added in #366

### CODE REVIEW

File: affiliated/registry.json [hunk 1]
```diff
@@ -994,7 +994,7 @@
           },
           {
             "name": "mocpy",
-            "maintainer": "Matthieu Baumann <matthieu.baumann@astro.unistra.fr> and Thomas Boch <thomas.boch@astro.unistra.fr>",
+            "maintainer": "François-Xavier Pineau <francois-xavier.pineau@astro.unistra.fr>, Matthieu Baumann <matthieu.baumann@astro.unistra.fr>, and Thomas Boch <thomas.boch@astro.unistra.fr>",
```
Comment (pllim) [inline affiliated/registry.json:997]: Actually, is this supposed to be alphabetical order?

Comment (pllim) [inline affiliated/registry.json:997]: Actually, is this supposed to be alphabetical order?

**Comment by pllim** [comment]
OK, let's wait to see if mocpy devs have anything to say about this.

**Comment by adrn** [comment]
@fxpineau and @bmatthieu3 - we need to get this in ASAP if you would like the updated reference to be included in the Astropy paper at submission. Could you please advise here?

**Comment by fxpineau** [comment]
@adrn, @pllim (cc @bmatthieu3 ) the order in the author list represents the current ability to actually maintain the project.
I think we are ok leaving the order as it is. Thank you very much.

**Comment by adrn** [comment]
Thanks for responding @fxpineau !

> I think we are ok leaving the order as it is.

Just to clarify: As it is in this pull request? Or did you want to close this pull request and keep the file as it is in the repo?

**Comment by fxpineau** [comment]
@adrn 

> Just to clarify: As it is in this pull request?

Sorry. Yes: "As it is in this pull request".
I.e: 
> "maintainer": "François-Xavier Pineau <francois-xavier.pineau@astro.unistra.fr>, Matthieu Baumann <matthieu.baumann@astro.unistra.fr>, and Thomas Boch <thomas.boch@astro.unistra.fr>",

Thank you again

**Comment by pllim** [comment]
OK I think I fixed it. Maybe we can just use the Squash and Merge button when this is approved? Thanks!

**Comment by adrn** [comment]
Thanks @pllim! I think we're just waiting on @bmatthieu3 to approve this, and then we can get this in.

**Comment by bmatthieu3** [comment]
@adrn, @pllim, this sounds good to me as well. Thanks

**Comment by pllim** [comment]
Thanks, all!

**Pull request merged**